frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_action.dart

```diff
@@ -27,6 +30,8 @@ enum SmartEditAction {
         return 'Correct this to standard English:\n\n$input';
       case SmartEditAction.improveWriting:
         return 'Rewrite this:\n\n$input';
+      case SmartEditAction.makeItLonger:
+        return 'Make this text longer:\n\n$input';
```

Comment:
By the way, have you tested this prompt?

Reply:
Tested it, looks good to me